plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -76,6 +79,30 @@ add_ovs_flows() {
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=0, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=${tenant_id}, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
     fi
+
+    # Pod ingress == OVS vSwitch egress
+    # linux-htb used here since that's the Kubernetes default traffic shaper too
+    if [ -n "${ingress_bw}" ]; then
+        qos=$(ovs-vsctl create qos type=linux-htb other-config:max-rate=${ingress_bw})
+        ovs-vsctl set port ${veth_host} qos=${qos}
+    fi
+
+    # Pod egress == OVS vSwitch ingress
+    if [ -n "${egress_bw}" ]; then
+        # OVS ingress_policing specified in Kbps
+        egress_kbps=$((${egress_bw}/1000))
+        ovs-vsctl set interface ${veth_host} ingress_policing_rate=${egress_kbps}
+
+        # OVS also recommends that the burst be the greater of the interface
+        # MTU or 1/10th the ingress_policing_rate
```

Comment:
Kubernetes just sends the rate to the 'tc' tool, and it does some magic and figures things out.  Turns out OVS does some defaults magic and sets the burst to something sane too.  So I don't think we actually need to bother with the burst rate.  See https://github.com/osrg/openvswitch/blob/master/lib/netdev-linux.c#L2612 for what OVS is doing.